docs/ref/models/database-functions.txt

```diff
@@ -1147,18 +1147,20 @@ Returns a random value in the range ``0.0 ≤ x < 1.0``.
 ``Round``
 ---------
 
-.. class:: Round(expression, **extra)
+.. class:: Round(expression, precision=0, **extra)
 
 Rounds a numeric field or expression to the nearest integer. Whether half
-values are rounded up or down depends on the database.
+values are rounded up or down depends on the database. The ``precision``
```

Comment:
_"Rounds a numeric field or expression to the nearest integer."_ is outdated. I pushed edits.